examples/framework-examples/vue-example/cli-applet/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "vue-example",
+  "main": "dist/index.html",
```

Comment:
Yes. This is definitely "good to know" how to adjust the main file (entrypoint) of the application for the applet environment.
However, by the sigangeOS CLI, the dist/index.html is the default value. So this change is not necessary.